lib/knapsack_pro/client/api/v1/queues.rb

```diff
@@ -17,6 +17,8 @@ def queue(args)
                 :node_index => args.fetch(:node_index),
                 :node_build_id => KnapsackPro::Config::Env.ci_node_build_id,
                 :user_seat => KnapsackPro::Config::Env.masked_user_seat,
+                :build_author => KnapsackPro::RepositoryAdapters::GitAdapter.new.build_author,
+                :commit_authors => KnapsackPro::RepositoryAdapters::GitAdapter.new.commit_authors,
```

Comment:
We could do a performance improvement here and move the lines down below. I completely forgot about it. Just noticed it now.

We can send build author and commit authors only in the request that supposed to initialize a new queue. Thanks to that we don't have to transfer unnecessary data over the network for each Queue API request that is consuming tests from the Queue. Similarly like we do for test files. 

Additional benefit of that approach is that we don't have to call git command on each request. I guess this could cause a small delay in running tests if you call git many times on each CI node for each request.





```ruby
            if request_hash[:can_initialize_queue] && !request_hash[:attempt_connect_to_queue]
                request_hash.merge!({
                  :test_files => args.fetch(:test_files),
                  :build_author => KnapsackPro::RepositoryAdapters::GitAdapter.new.build_author,
                  :commit_authors => KnapsackPro::RepositoryAdapters::GitAdapter.new.commit_authors
                })
            end
```